cli/command/image/build_linux_test.go

```diff
@@ -0,0 +1,54 @@
+//+build linux
```

Comment:
Could we use `skip` instead of the build directive? Or does this not build at all on windows?

Reply:
@dnephin it doesn't build at all on windows — there was a skip before 😓